README.rst

```diff
@@ -227,6 +227,31 @@ Installing the latest master branch of Salt:
   curl -L https://bootstrap.saltstack.com | sudo sh -s -- git master
 
 
+Install on Windows
+~~~~~~~~~~~~~~~~~~
+
+Using ``PowerShell`` to install latest stable version:
+
+.. code:: console
+
+  Invoke-WebRequest `
+    -Uri https://winbootstrap.saltstack.com
```

Comment:
missing ` on the end of this line.

Reply:
Thanks I checked Invoke-WebRequest syntax and fixed that line.